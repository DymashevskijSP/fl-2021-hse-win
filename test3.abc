Sigma:"a,b,c"
Q:"q_0,q_1,q_2"
q0:"q_0"
Terminals:"q_0"
delta: '(q_0, a)->q_0, (q_0, b)->q_0, (q_0, c)->q_1, (q_1, a)->q_2, (q_1, b)->q_2, (q_1, c)->q_0, (q_2, a)->q_2, (q_2, b)->q_2, (q_2, c)->q_2'
