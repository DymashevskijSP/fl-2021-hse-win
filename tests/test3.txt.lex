LexToken(RULENAME,'A',1,0)
LexToken(EQ,'=',1,1)
LexToken(LBRACKET,'[',1,2)
LexToken(PLAINTEXT,'a',1,3)
LexToken(RBRACKET,']',1,6)
LexToken(COMMA,',',1,7)
LexToken(LBRACE,'{',1,8)
LexToken(PLAINTEXT,'b',1,9)
LexToken(RBRACE,'}',1,12)
LexToken(COMMA,',',1,13)
LexToken(INDENT,1,2,15)
LexToken(LBRACKET,'[',2,19)
LexToken(PLAINTEXT,'c',2,20)
LexToken(RBRACKET,']',2,23)
