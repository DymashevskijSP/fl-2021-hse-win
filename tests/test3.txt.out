Node:   A =   [ a ] ,   { b } ,  [ c ]: Children  A =   [ a ] ,   { b } ,  [ c ] |
Node:  A =   [ a ] ,   { b } ,  [ c ]: Children A | = |   [ a ] ,   { b } ,  [ c ] |
Node: A: Children EMPTY
Node: =: Children EMPTY
Node:   [ a ] ,   { b } ,  [ c ]: Children  [ a ] | , |   { b } ,  [ c ] |
Node:  [ a ]: Children [ | a | ] |
Node: ,: Children EMPTY
Node:   { b } ,  [ c ]: Children  { b } | , |  [ c ] |
Node: [: Children EMPTY
Node: a: Children EMPTY
Node: ]: Children EMPTY
Node:  { b }: Children { | b | } |
Node: ,: Children EMPTY
Node:  [ c ]: Children [ | c | ] |
Node: {: Children EMPTY
Node: b: Children EMPTY
Node: }: Children EMPTY
Node: [: Children EMPTY
Node: c: Children EMPTY
Node: ]: Children EMPTY